core/classes/Core/User.php

```diff
@@ -181,11 +181,14 @@ public function addGroup(int $group_id, int $expire = 0, $group_data = null): bo
             $this->_groups[$group_id] = new Group($group_data);
         }
 
+        $default_language = new Language('core', DEFAULT_LANGUAGE);
         EventHandler::executeEvent('userGroupAdded', [
             'username' => $this->data()->username,
             'user_id' => $this->data()->id,
             'group_id' => $group_id,
             'group_name' => $this->_groups[$group_id]->name,
+            'avatar_url' => AvatarSource::getAvatarFromUserData($this->data()),
```

Comment:
You are missing the full url param, anyways you should use `$user->getAvatar(128, true),` instead